CONTRIBUTING.md

```diff
@@ -66,14 +72,40 @@ Pull requests should have a title that follows the specification, otherwise, mer
 - `refactor: ` prefix in the title indicates that PR is only related to refactoring and there is no need to trigger release.
 
-What about MAJOR release? just add `!` to the prefix, like `fix!: ` or `refactor!: `
+What about a MAJOR release? Just add `!` to the prefix, like `fix!: ` or `refactor!: `
 
-Prefix that follows specification is not enough though. Remember that the title must be clear and descriptive with usage of [imperative mood](https://chris.beams.io/posts/git-commit/#imperative).
+A prefix that follows specification is not enough. Remember that the title must be clear and descriptive with the usage of [imperative mood](https://chris.beams.io/posts/git-commit/#imperative).
 
-Happy contributing :heart:
+Happy contributing! :heart:
 
 ## License
 When you submit changes, your submissions are understood to be under the same [Apache 2.0 License](https://github.com/asyncapi/asyncapi/blob/master/LICENSE) that covers the project. Feel free to [contact the maintainers](https://www.asyncapi.com/slack-invite) if that's a concern.
 
-## References
-This document was adapted from the open-source contribution guidelines for [Facebook's Draft](https://github.com/facebook/draft-js/blob/master/CONTRIBUTING.md).
\ No newline at end of file
+## Contribution recognition
+
+We use the [All Contributors](https://allcontributors.org/docs/en/specification) specification to handle recognitions. For more details read the [`recognize contributors` document](https://github.com/asyncapi/community/blob/master/recognize-contributors.md).
+
+## Maintainers setup
+
+To streamline project management and facilitate onboarding, the maintainer's set up in the project includes two roles: `triager` and `committer`.
+
+There are also two separate areas of responsibility: docs and code. There can be a committer responsible for code only, and there can also be a triager responsible for docs only. There can also be a maintainer that holds all roles and participages in different areas.
+
+We recognize that because of the size of the project and its complexity, areas of responsibility can also get more granular. For example, there can be a committer responsible for docs but only community docs or a committeer responsible only for project design. These exceptions are assessed and approved by project maintainers.
+
+### Triager
+
+Triagers are responsible for labeling, commenting, and assisting with issue and pull request management.
+
+- Triagers assess newly-opened issues and pull requests.
+- Responsibilities include labeling issues and pull requests, commenting, closing, and reopening items as needed, and assisting users and novice contributors.
+- Triagers play a crucial role in enforcing the contributor guide and maintaining a clean backlog.
+- If a triager plans to become a committer, they should consult with existing committers to gradually gain more rights.
```

Comment:
What should be the criteria for a triager to become a committer?

Reply:
I guess doing the work of a committer for some time?

not sure we are able to define, in the end someone has to gain trust of committer to become one - committer needs to be comfortable with another person to approve and merge

so committer, in theory knows triager from their work, and more or less know what knowledge they are missing and what they need to work on - help define next steps 🤔 

not sure if we are able to be more specific?

Reply:
I think the clear point to become a committer from triager, is to take their stance, opinions or give suggestions on some of the critical issues in the repo, or being the part/mentor of any big/critical project idea inside `website`. In that way, other committers can find enough skills inside the required triager to become a committer. Moreover, the activity of triager also determines whether the person can be a active committer or not.